packages/reactor-browser/src/hooks/add-ph-event-handlers.ts

```diff
@@ -81,6 +83,8 @@ const phGlobalEventHandlerRegisterFunctions: PHGlobalEventHandlerAdders = {
   selectedNodeId: addSelectedNodeIdEventHandler,
   vetraPackages: addVetraPackagesEventHandler,
   selectedTimelineRevision: addSelectedTimelineRevisionEventHandler,
```

Comment:
I guess `selectedTimelineRevision` and `selectedTimelineItem` are the same thing, right @ryanwolhuter?